src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ExcelAdaptorServiceImpl.java

```diff
@@ -70,9 +70,6 @@
     public <T> List<T> parseExcel(Workbook workbook, Class<T> classType) {
         evaluator = workbook.getCreationHelper().createFormulaEvaluator();
         List<String> validHeaders;
-        if (workbook.getNumberOfSheets() < 1) { // check at least 1 sheet present
```

Comment:
Hope that this is being checked elsewhere.

Reply:
yes, in here - https://github.com/hmcts/rd-caseworker-ref-api/blob/master/src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ExcelAdaptorServiceImpl.java#L84